docs/guide/docs/infrastructure/cluster-digital-ocean.md

```diff
@@ -1,32 +1,30 @@
 ---
-id: cluster-digital-ocean
+id: digital-ocean
 title: Deploying a cluster on Digital Ocean
 ---
 
 ## Prerequisite
-
 - Create a domain name dedicated to your Botpress Cluster, and generate a certificate. If you don't have a certificate, you can [follow these instructions to generate one quickly using Let's Encrypt.](https://www.digitalocean.com/community/tutorials/how-to-use-certbot-standalone-mode-to-retrieve-let-s-encrypt-ssl-certificates-on-ubuntu-1804)
 
 ## Instructions
+1. [Create a new Load Balancer](https://docs.digitalocean.com/products/networking/load-balancers/how-to/create/#:~:text=Setting%20up%20a%20load%20balancer,the%20Load%20Balancers%20overview%20page.)
 
-1. Create a new Load Balancer
-
-   - Forwarding Rules: `HTTPS 443` -> `HTTP 3000` (define the certificate created earlier). It is also possible to use Passthrough if you setup NGINX to redirect HTTPS requests
+   - Forwarding Rules: `HTTPS 443` -> `HTTP 3000` (define the pre-requisite certificate created earlier). It is also possible to use [Passthrough](https://www.digitalocean.com/docs/networking/load-balancers/how-to/ssl-passthrough/) if you setup NGINX to redirect HTTPS requests
    - Health Checks: You might want to set those numbers lower
    - Sticky Sessions: `Enabled`
 
-2. Create a droplet (_Ubuntu 18.04_) for your [Redis instance](https://www.digitalocean.com/community/tutorials/how-to-install-and-secure-redis-on-ubuntu-18-04)
+2. Create a droplet (_Ubuntu 20.04_) for your [Redis instance](https://www.digitalocean.com/community/tutorials/how-to-install-and-secure-redis-on-ubuntu-20-04)
```

Comment:
have you tried this ? we don't have the native extensions built for 20-04. our latest native ext are 18.04 .  I personally didn't try it out so I can't guarantee this works properly

Reply:
@franklevasseur do our native extensions work on 20.04 ?

Reply:
They weren't built on 20.04, but the default ones were built and on 18.04 and it works fine. I've been using ubuntu 20.04 on wsl for a while now and I have no problem

Reply:
I got the heads up that 20.04 works from @spg so I am sure we should be fine here.